src/components/pages/edit-doc.js

```diff
@@ -333,13 +335,19 @@ function EditDoc({ initialState, PageTemplate }) {
     handleClick: handleCancelClick
   });
 
-  if (proposedSectionKeys.length) {
-    alerts.push({
-      message: t('proposedSectionsAlert'),
-      type: ALERT_TYPE.info,
-      showInFullScreen: false
-    });
-  }
+  useEffect(() => {
+    const newAlerts = [...globalAlerts];
+
+    if (initialState.proposedSections?.length) {
```

Comment:
I've changed that to rely on the `initialState.proposedSections` instead of `[state.]proposedSectionKeys.length` from before, which from a "business logic" perspective means that:
- before, the alert was shown for a clone document once a section was selected to be applied on the new document
- now, the alert is shown for a cloned document that has any sections

To me this makes more sense, given that the alert message is generic:
"Apply or discard the proposed sections. Unapplied sections are only visible until you leave the page."

What do you think?

Reply:
I didn't quite get the comment, but the behavior now - as far as I can see - is identical to back when the page was a class component (which is what we want).